src/reducers/userData.js

```diff
@@ -0,0 +1,19 @@
+import * as types from '../actions/actionTypes';
+
+const INITIAL_STATE = { userData: {}, loading: false };
+
+export default function userData(state = { ...INITIAL_STATE }, action = {}) {
```

Comment:
Maybe it's better to call reducers like _user_ (or _currentUser_) instead of _userData_?